### PR TITLE
Update tiup-cluster-topology-reference.md

### PR DESCRIPTION
**Pull request by srstack**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md
Comment (lilin90) [inline tiup/tiup-cluster-topology-reference.md:38]:
```suggestion
- `enable_tls`：是否对集群启用 TLS。启用之后，组件之间、客户端与组件之间都必须使用生成的 TLS 证书进行连接，默认值：false
```

**Comment by shichun-0415** [comment]
@srstack Thanks for your contribution. FYI: GitHub does not support modifying the `status` label manually. We can only merge a PR after it obtains 2 LGTMs by using the `/merge` command. And only [docs committers](https://prow.tidb.net/tichi/repos/pingcap/docs-cn/pulls/11849/owners) can merge PRs in docs repo. We will merge it ASAP after it passes both tech review and editorial review.

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**
